lib/src/Status/OSNotification.cpp

```diff
@@ -158,6 +164,22 @@ void OSNotification::showNotification(const QString& title,
 
 #elif defined Q_OS_MACOS
     showNotificationMacOs(title, message, identifier);
+#elif defined Q_OS_LINUX
+    static QString notifyExe = QStandardPaths::findExecutable(QStringLiteral("notify-send"));
+    if (notifyExe.isEmpty()) {
+        qWarning() << "'notify-send' not found; OS notifications will not work";
+        return;
+    }
+
+    //qDebug() << "NOTIFY-SEND; title:" << title << "; message:" << message << "; id:" << identifier;
```

Comment:
@caybro is this needed?

Reply:
Nope, I can drop that, was just used when debugging this

Reply:
Done